content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -1004,14 +1004,13 @@ RFC3339 timestamp. Presence or absence of an expiry has the following impact:
   }
 }
 ```
-
-The plugin can optionally be called with an environment variable, `KUBERNETES_EXEC_INFO`,
-that contains information about the cluster for which this plugin is obtaining
-credentials. This information can be used to perform cluster-specific credential
-acquisition logic. In order to enable this behavior, the `provideClusterInfo` field must
-be set on the exec user field in the
-[kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/). Here is an
-example of the aforementioned `KUBERNETES_EXEC_INFO` environment variable.
+To provide the plugin with information about the cluster for which it is
+obtaining credentials, the `provideClusterInfo` field can be set on the exec user
+field in the [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+The plugin will then be supplied with an environment variable, `KUBERNETES_EXEC_INFO`.
+Information from this environment variable can be used to perform cluster-specific
+credential acquisition logic. Here is an example of the aforementioned
+`KUBERNETES_EXEC_INFO` environment variable.
```

Comment:
Hi @margocrawf . Thanks for opening this PR.
Based on the changes, here are some optional edits/suggestions:

To enable the exec plugin to obtain cluster-specific information, set `provideClusterInfo` on the `user.exec`
field in the [kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
The exec plugin reads the credential access logic stored in the `KUBERNETES_EXEC_INFO` environment variable.
The following `ExecCredential` manifest describes a cluster information sample:

Reply:
Hi @kbhawkey, thanks for the suggestions!

I like the change you made to the first sentence, I think the active voice makes it more clear. I can go ahead and make that change.

One thing that I found confusing when I read the original paragraph was understanding the cause-and-effect relationship between setting `user.exec.provideClusterInfo` and having access to the `KUBERNETES_EXEC_INFO` environment variable from in the exec plugin. My hope is that my second sentence makes that relationship more clear to readers. I worry that it is less clear in your revision because it jumps straight to what the exec plugin does with the environment variable.

I also like your rewording of the last sentence. I will incorporate that as well.

Reply:
Sure!